formula1_race_analysis/q1_report_generator.py

```diff
@@ -0,0 +1,50 @@
+import sys
+from pathlib import Path
+
+import click
+import structlog
+
+from formula1_race_analysis.custom_exceptions import CustomFileNotFoundError
+from formula1_race_analysis.logging_config import logger_factory
+from formula1_race_analysis.q1_session_analyzer import display_race_report, get_results_of_first_qualifying_session
+
+logger_factory()
+logger: structlog.stdlib.BoundLogger = structlog.get_logger()
+
+
+@click.command()
+@click.option("--data_dir", type=Path, required=True, help="[--asc | --des]")
+@click.option("--asc", "order", flag_value="asc", default=True, help="To sort results in ascending order")
```

Comment:
I think two order options can be squized into one
Try this out
https://click.palletsprojects.com/en/stable/options/#choice-options